apps/protocol-frontend/src/components/CreateDaoLayout.tsx

```diff
@@ -35,7 +33,6 @@ const CreateDaoLayout = () => {
         width="100%"
       >
```

Comment:
(aribtrary thread)

Tested "Change import status to Member not Recruit for users added directly via the Create a DAO or via DAO Settings" and my added user was still a recruit

Reply:
Looks like #398 fixes this, please change the description as it says it fixes it here.

Reply:
Ah, my mistake. I copied that over from the ticket and forgot to remove. Thanks!